### ISSUE TITLE
Can we add a DLM style search box to each page and have that display all the results?

### ISSUE THREAD

**Issue by allanchaplin** [nittygritty-uk/Atvero-Issues#344]
Rather than a search button in the command bar, use the DLM one and open the it with search box populated with the content from the start bar search and the correct tab already open.  So for DLM it would be:

![screenshot 2019-03-06 at 16 50 52](https://user-images.githubusercontent.com/53964/53898485-0e0f4380-4030-11e9-8757-8fdc8a4e67b5.png)

then you would see:

![screenshot 2019-03-06 at 16 51 03](https://user-images.githubusercontent.com/53964/53898495-14052480-4030-11e9-8a2a-c37632666501.png)

So that people would hopefully see the results they're most interested in first.

**Comment by LBoyle** [comment]
Yes, using the callback provided by the higher order component. Or we could add another one to run the search the same way.

**Comment by allanchaplin** [comment]
Don't do this one yet, I'm still thinking about it.
